rdkit/Chem/Draw/MolDrawing.py

```diff
@@ -416,27 +416,25 @@ def AddMol(self, mol, centerIt=True, molTrans=None, drawingTrans=None, highlight
           symbolLength = len(symbol)
         else:
           base = atom.GetSymbol()
-          if (base == 'H' and (iso == 2 or iso == 3) and
-              self.drawingOptions.atomLabelDeuteriumTritium):
+          if base == 'H' and (iso == 2 or iso == 3) and self.drawingOptions.atomLabelDeuteriumTritium:
```

Comment:
I comment on this just for the record. Long lines are ok if you have a large monitor and I also aim to avoid breaking if statements to two lines. However, in a collaborative code base, it's better to write code that doesn't get too wide. For example, I tend to work most of the time just on my laptop. 

Anyway, it's better to consider one of the automated formatting tools to make the code more consistent. I've used yapf on this code base a long time ago, but we didn't implement an automated process. This has nothing to do with this PR though.